server/src/db/index.js

```diff
@@ -0,0 +1,23 @@
+const { MongoClient } = require('mongodb');
+
+const { appConfig } = require('../config/connectionStringBuilder');
+
+let db = null;
+async function getDatabase() {
+  if (!db) {
+    const connectionDetails = appConfig;
+    const client = new MongoClient(connectionDetails.connectionURI, {
```

Comment:
How does this tie in with the other Mongo classes added in this PR? I think I'm missing something

Reply:
This connection config was using mongo driver directly instead of Mongoose. It's not used anymore and I removed it from files.